iroha_data_model/src/isi.rs

```diff
@@ -179,22 +182,22 @@ pub struct GrantBox {
 }
 
 /// Generic instruction to set value to the object.
-#[derive(Debug, Clone, Serialize, Deserialize, Encode, Decode)]
+#[derive(Debug, Clone, Serialize, Deserialize, Encode, Decode, Introspect)]
 pub struct Set<O>
 where
-    O: ValueMarker,
+    O: ValueMarker + Introspect,
 {
     /// Object to equate.
     pub object: O,
 }
 
 /// Generic instruction to set key value at the object.
-#[derive(Debug, Clone, Serialize, Deserialize, Encode, Decode)]
+#[derive(Debug, Clone, Serialize, Deserialize, Encode, Decode, Introspect)]
 pub struct SetKeyValue<O, K, V>
 where
-    O: Identifiable,
-    K: ValueMarker,
-    V: ValueMarker,
+    O: Identifiable + Introspect,
+    K: ValueMarker + Introspect,
+    V: ValueMarker + Introspect,
```

Comment:
Do we need `Introspect` bound here?

Reply:
Yes

Reply:
Also here it is not needed as we discussed during the meeting.